koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -537,6 +537,13 @@ def sigint_handler(*args, **kwargs):
 
         elif not has_succeeded:
             error_message = self.osbs().get_build_error_message(build_id)
+
+            if self.osbs().build_not_finished(build_id):
+                try:
+                    self.osbs().cancel_build(build_id)
+                except Exception:
+                    pass
```

Comment:
Should this be empty?

Reply:
well we won't see logs from koji-c anyway so it doesn't matter at all

Reply:
So if canceling the build raises an exception, we just ignore it and continue?
It looks to me like the exception is "handled" by falling through to `if error_message`. Do we have any guarantee that `get_build_error_message` will *never* come up empty...?

Reply:
1st I am getting original error message

then IF pipeline run isn't finished I cancel pipeline run (and I don't care anymore about it, I want to propagate only the original message, not that "additional" cancel failed)

Reply:
if error_message is empty that is handled bellow, but how is that relevant?

Reply:
I'm concerned about what happens if canceling the build fails with an exception

Reply:
build fails with the original error

Reply:
Could we log the exception anyway? Maybe we won't see it, but just out of principle.

Reply:
sure